md/bdm-in-rest-api.md

```diff
@@ -216,6 +216,108 @@ class CarManagement implements RestApiController {
     }
 ```
 
+### Returning the whole object with an API link load in the lazy fields
+
+The idea is to create a custom Json serializer.  
+A custom Json serializer is a class which extends *com.fasterxml.jackson.databind.JsonSerializer*. There is a method *serialize* to implement, which has the responsability to serialize the input model into Json.  
+The custom Json serializer has to come with an other classe, an object mapper,  which extends *com.fasterxml.jackson.databind.ObjectMapper*.  
+This mapper register a simple module (*com.fasterxml.jackson.databind.module.SimpleModule*), which has to contains the custom serializer.  
+At the end, in your rest API endpoint, you interact with the mapper.
+
+Here is an implementation example for the object Car which has four lazy attributes of type Wheel:  
+
+The serializer takes a list of Car in input, and build a Json object for each car. The wheels are replaced with links to an other Rest API extension with the car ID and the wheel number in parameter. Calling this API will return the wheel. This is a classic lazy behavior.
```

Comment:
I didn't notice it during my 1st review but you can make your serializer manage Car object instead of List of Car.
Jackson knows how to manage List and delegates serialization of list elements to their given serializer.

Reply:
Ho, i gonna test that